src/backend/InvenTree/company/serializers.py

```diff
@@ -166,6 +166,8 @@ def annotate_queryset(queryset):
 
     image = InvenTreeImageSerializerField(required=False, allow_null=True)
 
+    email = serializers.EmailField(required=False, default='', allow_blank=True)
```

Comment:
Q (Non blocking): Is this related to revisions?

Reply:
No, looks like these have bled through from my attempts to support the new django-rest-framework requirements. Not essential here, but doesn't hurt either